rocketpy/motors/Tank.py

```diff
@@ -0,0 +1,524 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Giovani Hidalgo Ceotto, Oscar Mauricio Prada Ramirez, João Lemes Gribel Soares, Mateus Stano, Pedro Henrique Marinho Bressan, Patrick Bales, Lakshman Peri, Gautam Yarramreddy, Curtis Hu, and William Bradford"
+__copyright__ = "Copyright 20XX, RocketPy Team"
+__license__ = "MIT"
+
+from abc import ABC, abstractmethod
+
+from rocketpy.Function import Function, funcify_method
+from rocketpy.utilities import except_negative
+
+
+class Tank(ABC):
+    def __init__(self, name, geometry, gas, liquid=0):
+        self.name = name
+        self.geometry = geometry
+        self.gas = gas
+        self.liquid = liquid
```

Comment:
The argument name "mass" makes it seem like it is the tank mass, but it is only the mass of the tank's contents. I think we should change this name for something a bit more descriptive

Reply:
Agreed.

Reply:
@phmbressan , any opinion in this one?

Reply:
Good points. I believe the name `fluidMass` is more descriptive.

Reply:
Solved here: [c810266](https://github.com/RocketPy-Team/RocketPy/pull/233/commits/c81026638be688442ae06f8056b4c96552b422e7)